src/components/layouts/Footer.jsx

```diff
@@ -0,0 +1,9 @@
+const Footer = () => {
+  return (
+    <footer className="flex w-screen  flex-row flex-wrap items-center justify-center bg-lightGreen w-full h-24 text-center text-black bottom-0">
```

Comment:
Does the properties 'flex-wrap' and 'text-center' are necessary in the footer? 
Asking because 'items-center justify-center ' already centered all that right now inside the footer. And why the app needs ''flex-wrap'' property right now? 

You did a great job positioning the footer!

Reply:
removed flex-wrap and text center. Items-center is enough. Thank you

Reply:
bottom0 removed

Reply:
Oh no! Tanya< could you please return 'text center' property. I realizes why you pot it in the code:

with 'text center'
<img width="147" alt="CTD Practicum" src="https://github.com/user-attachments/assets/7eccb56c-455b-4068-8f60-abce2348a777">

without 'text center'
<img width="154" alt="CTD Practicum" src="https://github.com/user-attachments/assets/566dee94-b43f-486b-97b8-31fe815fa666">